api/serializers/student/dashboard.py

```diff
@@ -88,3 +89,21 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
             raise ConflictError(f"Student {self.instance.pk} must be in group {from_group.pk}")
 
         return validated_attrs
+
+
+class DashboardStudentMissedClassSerializer(serializers.Serializer[Any]):
+    group_id = serializers.IntegerField()
+    notified = serializers.BooleanField()
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        try:
+            group = Group.objects.get(pk=int(attrs["group_id"]))
+        except Group.DoesNotExist:
+            raise ConflictError(f"Group {attrs['group_id']} not found")
```

Comment:
Is it really ConflictError? Maybe some other type of exception?

Reply:
We have discussed it at some point I think so if we use some param as a part of the URL (`api/v2/user/<id>`) then it might be a good idea to use not found, on the contrary, if something inside a body it means that we found a source but due to some conflicts were unable to return it. At least I  follow this logic

Reply:
But here we didn't really find anything, so it seems that NotFound is more suitable. But if in other places we use ConflictError for such cases - let's keep it as it is

Reply:
Let me interfere here @Roxe322 , we had this discussion with you and looked at a huge thread on SO. The main argument against 404 is that in our case the address is valid and the server can respond correctly (just has nothing to show), while 404 gives the signal that the URL is completely wrong, no resource was found.